src/qt/rpcconsole.cpp

```diff
@@ -730,9 +730,15 @@ void RPCConsole::clear(bool clearHistory)
             ).arg(fixedFontInfo.family(), QString("%1pt").arg(consoleFontSize))
         );
 
+#ifdef Q_OS_MAC
+    QString clsKey = "<b>⌘cmd-L</b>";
```

Comment:
Can you please move this HTML down to `arg(clsKey)`? E.g.
```C++
arg("<b>"+clsKey+</b>)
```
or something and remove it from here?